### PR TITLE
FastTimerServiceClient bug fix

### PR DESCRIPTION
**Pull request by mtosi**
bug fix
plots vs lumi were empty in runs at the beginning of the fill

### CODE REVIEW

**Comment by mtosi** [comment]
@cmsbuild, please test

**Comment by Martin-Grunewald** [comment]
@mtosi 
92X backport needed as well...

**Comment by mtosi** [comment]
I included it in the PR https://github.com/cms-sw/cmssw/pull/20695
I hope it is fine

**Comment by Martin-Grunewald** [comment]
+1

File: HLTrigger/Timer/plugins/FastTimerServiceClient.cc [hunk 1]
```diff
@@ -418,8 +418,8 @@ FastTimerServiceClient::fillPlotsVsLumi(DQMStore::IBooker & booker, DQMStore::IG
     label.erase(label.find("_byls"));
 
     MonitorElement* me = getter.get(current_path + "/" + m);
-    double ymin        = me->getTProfile()->GetMinimum();
-    double ymax        = me->getTProfile()->GetMaximum();
+    float ymin        = 0.;
+    float ymax        = std::numeric_limits<float>::max();
```
Comment (davidlange6) [inline HLTrigger/Timer/plugins/FastTimerServiceClient.cc]: hi @mtosi - naively it looks to be better just to drop the ymin and ymax arguments from the bookProfile call below. Why keep them?

**Comment by mtosi** [comment]
I tried, but I got compilation errors
because apparently the bookProfile methods do not handle such a possibility
https://github.com/mtosi/cmssw/blob/54acd90ac2532e910b2127c565c72ea1f3e76cee/DQMServices/Core/interface/DQMStore.h#L435-L476
  mia

**Comment by davidlange6** [comment]
sigh - indeed - maybe @dmitrijus  can eventually fix that

**Comment by davidlange6** [comment]
+1

**Pull request merged**